traitsui/examples/demo/Extras/Image_editor_demo.py

```diff
@@ -15,20 +15,18 @@
 from traits.api \
     import HasTraits, Str
 
-from traitsui.api \
-    import View, VGroup, Item
+import traitsui.api
 
 from traitsui.api \
-    import ImageEditor
+    import View, VGroup, Item, ImageEditor
 
 from pyface.image_resource \
     import ImageResource
 
 # Constants:
 
 # Necessary because of the dynamic way in which the demos are loaded:
```

Comment:
If I run the script from the command line, even just `search_path = []` would work, but that's using some default magic (with inspection on where the code is running from etc.). I believe this is the magic:
https://github.com/enthought/traits/blob/a0d0a70331ec202eb33b6439570175c869fe353c/traits/trait_base.py#L256-L305

That would not work when this example is run from within the demo application. `join(dirname(__file__))` remove the assumption and dependency on the magic: It is more explicit, which I think is a good thing to do anyway. Now the comment makes it look worse than this actually is.

What do you think about rewording the comment to something along the line of `# The images folder is in the same folder as this file`?